app/models/workers_scaling/algorithm.rb

```diff
@@ -70,16 +77,23 @@ def initial_deployment
     ##
     # Main algorithm loop, executed on specified event (e.g. finished simulation) or
     # when given time since last execution passed. Should contain main algorithm logic.
-    def experiment_status_check
+    def execute_algorithm_step
       raise NOT_IMPLEMENTED
```

Comment:
Można zmienić na: http://stackoverflow.com/questions/13668068/how-to-signal-not-implemented-yet
Takie podejście jest stosowane np w fasadach, przy okazji wyeliminuje stałą NOT_IMPLEMENTED